source/_integrations/otbr.markdown

```diff
@@ -12,6 +12,19 @@ ha_config_flow: true
 ---
 
 The Open Thread Border Router integration allows calling an Open Thread Border Router's REST API from Python and via WebSocket.
-The integration is automatically set up when the "Silicon Labs Multiprotocol" add-on is installed.
+
+This integration is installed automatically when the **Open Thread Border Router** add-on is installed. This happens when you do one of the following procedures:
+
+- [Enabling Thread on Home Assistant Yellow](https://yellow.home-assistant.io/procedures/enable-thread/)
+- [Enabling Thread on Home Assistant SkyConnect](https://skyconnect.home-assistant.io/procedures/enable-thread/)
+- [Adding a HomeKit device via Thread](https://www.home-assistant.io/integrations/homekit_controller/)
+- [Adding a Matter device via Thread](https://www.home-assistant.io/integrations/matter/#adding-a-matter-device-to-home-assistant)
```

Comment:
When adding Thread devices via 3rdparty border routers the OTBR integration won't be used.

Just remove these two, the main gist is that this happens when installing the OTBR add-on.

```suggestion
```